package-manifest.toml

```diff
@@ -148,20 +127,25 @@ switch_variant = "stub"
 #   1c. cargo xtask dist -o
 # 2. Copy dendrite.tar.gz from dendrite/out to omicron/out
 # 3. Use type = "manual" instead of the "prebuilt"
```

Comment:
```suggestion
# 3. Use source.type = "manual" instead of "prebuilt"
```

Reply:
Done in 3b469a56d646e2d827d6b880c0db84e0b65537c1